lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,33 @@
+[Gitlab Generic Packages Repository](https://docs.gitlab.com/ee/user/packages/generic_packages/) supports looking up package versions from all type of packages registry supported by Gitlab and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+
+To specify which specific repository should be queried when looking up a package, the `registryUrl` has to be set like this: `https://gitlab.com/user/project`.
+As an example, `https://gitlab.com/gitlab-org/ci-cd/package-stage/feature-testing/new-packages-list` would look for packages in the generic packages repository of the `gitlab-org/ci-cd/package-stage/feature-testing/new-packages-list` project.
+
+**Usage Example**
+
+A real world example for this specific datasource would be maintaining packages versions in a config file.
```

Comment:
```suggestion
A real-world example for this specific datasource would be maintaining packages versions in a config file.
```

I'm not sure which is best here:

- `real world`
- `real-world`

Can a native English speaker decide which is best? 😉

Reply:
Mmh, I'm not native, but according to [Qora](https://www.quora.com/Should-real-world-be-hyphenated#:~:text=The%20simplest%20answer%20you%20can,%2C%20pronouns%20or%20noun%20phrases.%5D), in this case it's real-world.